### ISSUE TITLE
[BUG] [v2.10] Failed to install system chart fleet: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index

### ISSUE THREAD

**Issue by cpinjani** [rancher/rancher#48413]
**Rancher Server Setup**
- Rancher version: `v2.10-87b97590837835650d045937b669ee0206604758-head`
- Installation option:
   - Helm Chart, Kubernetes Cluster and version (k3s): v1.30.5+k3s1

**User Information**
- What is the role of the user logged in? Admin

**Describe the bug**
After installing Rancher, observing error: Failed to install system chart fleet: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index
Probably [build.yaml](https://github.com/rancher/rancher/blob/release/v2.10/build.yaml#L5) needs to be updated.

**Expected Result**
Fleet chart should be installed.

**Logs**
```
W1213 05:37:14.179888      39 warnings.go:70] v1 ComponentStatus is deprecated in v1.19+
2024/12/13 05:37:14 [INFO] Watching metadata for upgrade.cattle.io/v1, Kind=Plan
2024/12/13 05:37:15 [ERROR] Failed to install system chart fleet: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index
2024/12/13 05:37:15 [ERROR] Failed to install system chart fleet-crd: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index
2024/12/13 05:37:15 [ERROR] Failed to install system chart fleet: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index
2024/12/13 05:37:15 [ERROR] Failed to install system chart fleet-crd: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index
2024/12/13 05:37:15 [ERROR] Failed to install system chart fleet-crd: specified version 105.0.2+up0.11.2-rc.2 doesn't exist in the index
```

**Comment by cpinjani** [comment]
Fixed on `v2.10-ca82c9717ceb8ad2d0cc54f584d8adbae41a2341-head`, closing issue.
```
NAME        CHART       VERSION            RELEASE NAME   RELEASE VERSION   STATUS
fleet       fleet       105.0.2+up0.11.2   fleet          5                 deployed
fleet-crd   fleet-crd   105.0.2+up0.11.2   fleet-crd      1                 deployed
```

**Comment by mmartin24** [comment]
Verified after Github action process ended and Docker images where built Fleet is correctly deployed: 
https://github.com/rancher/fleet-e2e/actions/runs/12313608327

![image](https://github.com/user-attachments/assets/a55e82cf-5a3a-4e20-b0a6-59af48c31b63)

PS: this happens sometimes. If you run a test right in the moment the image is being built nothing is seen.
